text/0000-gps_support.md

```diff
@@ -0,0 +1,31 @@
+# NavSatFix (GPS) Support in Cartographer ROS
+
+## Summary
+[summary]: #summary
+
+The goal is to add [NavSatFix](http://docs.ros.org/api/sensor_msgs/html/msg/NavSatFix.html)  (GPS) support to Cartographer ROS. 
+The NavSatFix (GPS) data will be used in the pose graph optimization, using the fixed frame pose feature of Cartographer.
+
+## Motivation
+[motivation]: #motivation
+
+In outdoor applications, especially cars, GPS data is commonly available. 
+Using this data will reduce the drift in location, especially over large distances. 
+
+## Approach
+[approach]: #approach
+
+We will wire up the [sensor_msgs/NavSatFix](http://docs.ros.org/api/sensor_msgs/html/msg/NavSatFix.html) and introduce two new options for this: `use_nav_sat` and `fixed_frame_pose_sampling_ratio`.
+
+These messages will be [translated](https://en.wikipedia.org/wiki/Geographic_coordinate_conversion#From_geodetic_to_ECEF_coordinates) into a common cartesian coordinate frame, [ECEF](https://en.wikipedia.org/wiki/ECEF), and given to Cartographer as `FixedFramePoseData`.
+
+Alternatively, we could translate to a local ground reference frame ([ENU](https://en.wikipedia.org/wiki/Axes_conventions#Ground_reference_frames:_ENU_and_NED)) with the z-axis pointing up, and the origin at the start position.
```

Comment:
Could we settle on either ECEF or ENU? Maybe discuss the differences and advantages of both here, since both choices are rather opaque without more background.